lib/api/dsl/storage/index.js

```diff
@@ -370,7 +384,7 @@ function addConditions(hashKey, sfObj, condObj, index, collection, subfilter) {
       }
     }
     else {
-      keyword = Object.keys(cond).filter(k => k !== 'not')[0];
+      const keyword = Object.keys(cond).filter(k => k !== 'not')[0];
 
       condObj[index][collection][cId] = new Condition(cId, sfObj, cond.not ? 'not' + keyword : keyword, cond[keyword]);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 119 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)